backend/siarnaq/api/teams/views.py

```diff
@@ -173,3 +175,25 @@ def avatar(self, request, pk=None, *, episode_id):
             titan.upload_image(avatar, profile.get_avatar_path())
 
         return Response(None, status=status.HTTP_204_NO_CONTENT)
+
+    @action(
+        detail=False,
+        methods=["post"],
+        serializer_class=None,
+        permission_classes=(IsAuthenticated, IsEpisodeAvailable),
+    )
+    def record(self, request, pk=None, *, episode_id):
+        """Retrieve the win/loss record of a team"""
+        team = request.data["id"]
+        matches = Match.objects.filter(participants__team=team)
+
+        matches.annotate(
+            score_1=Max("participants__score", filter=Q(participants__team=team)),
+            score_2=Max("participants__score", filter=~Q(participants__team=team)),
```

Comment:
score_1 works properly but score_2 returns None always even when its not suppose too.

Reply:
Have you followed Jerry's suggestion regarding using `matches = matches.annotate(...)`?

Reply:
Yeah, I just haven't pushed yet.

Reply:
Does score_2 still not work?